### PR TITLE
Add ability to select specific coords

### PR DESCRIPTION
**Pull request by arnasbr**
Added the option to specify exact coordinates, instead of using the ones in `common.js`

Also some small code cleanup (removed unused vars)

### CODE REVIEW

File: scripts/common.js [hunk 1]
```diff
@@ -94,6 +94,19 @@ export function setThresholdsForScenarios (options) {
...
 }
 
+export function getCountryCoordinates (countryCode, coordinatesEnv, useProto = false) {
+  const data = useProto ? protoCountries : countries
+
+  if (coordinatesEnv) {
+    const coords = coordinatesEnv.split(',')
+    if (coords.length === 2) {
+      const [lat, lng] = coords.map(Number)
+      return { lat, lng }
```
Comment (danielnaumau) [inline scripts/common.js:104]: hmm. I'm not sure here. Is it the same as {lat: lat, lng: lng} ?

Comment (arnasbr) [reply scripts/common.js:104]: image
logged request body with -e COORDINATES=-8,13

File: README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ docker run
     -e API_KEY={API_KEY}
     -e HOST=api-dev.traveltimeapp.com //optional
     -e COUNTRY=gb //optional
+    -e COORDINATES=-8,13 // optional, this will overwrite COUNTRY env var
```
Comment (PKJonas) [inline README.md:23]: Will it actually overwrite the COUNTRY env var? That var is also used to construct the service url, not just to pick the coordinates.

File: scripts/common.js (hunk 1 above)
Comment (arnasbr) [reply scripts/common.js:104]: image
and this is without the new env var

Comment (danielnaumau) [reply scripts/common.js:104]: fair enough, just making sure

File: README.md (hunk 2 above)
Comment (arnasbr) [reply README.md:23]: Can you specify the exact location where COUNTRY var is used for that? As far as I can see, it's not even used in `generageBody`

Comment (PKJonas) [reply README.md:23]:
```
  const country = __ENV.COUNTRY || 'uk'
  ...
  const query = __ENV.QUERY || `api/v2/${countryCode(country)}/time-filter/fast/${transportation}`
  ...
  const countryCoords = protoCountries[country]
```

Comment (arnasbr) [reply README.md:23]: ah nvm, I see, proto is a bit different, I'll take a look

Comment (arnasbr) [reply README.md:23]: Tested it, it seems like the country that's specified in the url for proto requests makes no difference for the calculation. It's possible to use UK coords with any other country in the url, the result is still the same. I could mention this in README or comments if you think it's worth doing so

**Comment by arnasbr** [comment]
@PKJonas let me know if you see any more problems, when you have the time

**Review by PKJonas — APPROVED** [review]

File: scripts/common.js [hunk 3]
```diff
@@ -94,6 +94,36 @@ export function setThresholdsForScenarios (options) {
...
 }
 
+function coords (coordinatesEnv) {
+  const coords = coordinatesEnv.split(',').map(coord => coord.trim())
+  if (coords.length === 2) {
+    const [lat, lng] = coords.map(Number)
+    return { lat, lng }
+  } else {
+    console.error('Invalid coordinates format')
+    return null
```
Comment (danielnaumau) [inline scripts/common.js:104]: it's better to raise error here

**Pull request merged**